app/cdap/components/GridTable/index.tsx

```diff
@@ -263,6 +266,11 @@ export default function GridTable() {
           setLoading={setLoading}
           columnData={headersNamesList}
           missingDataList={dataQuality}
+          applyTransformation={(prepareDirective, column) => {
+            setColumnSelected(column);
+            setDirectiveFunction('');
```

Comment:
do we need to have some defualt value here? just checking so the app doesn't break when we click submit with empty string, check if possible, based on usecase